### PR TITLE
LWDEV-7986 fix logging

### PR DESCRIPTION
**Pull request by rocketmannot**

### CODE REVIEW

File: src/dist/cfg/log4j.properties [hunk 1]
```diff
@@ -6,6 +6,19 @@ log4j.appender.file.File=../log/server.log
 log4j.appender.file.layout=org.apache.log4j.PatternLayout
 log4j.appender.file.layout.ConversionPattern=%d{dd-MM HH:mm:ss:SSS} %p [%t] [%c{1}] %m%n
 
+log4j.appender.console=org.apache.log4j.ConsoleAppender
+log4j.appender.console.layout=org.apache.log4j.PatternLayout
+log4j.appender.console.layout.ConversionPattern=%d{dd-MM HH:mm:ss:SSS} %p [%t] [%c{1}] %m%n
+
+log4j.logger.AppStarter=error, console, file
```
Comment (barsukovskii) [inline src/dist/cfg/log4j.properties]: error -> info

File: src/dist/cfg/log4j.properties [hunk 2]
```diff
@@ -6,6 +6,19 @@ log4j.appender.file.File=../log/server.log
...
+log4j.additivity.AppStarter=false
+
+log4j.logger.com.lykke.matching.engine.utils.order.MinVolumeOrderCanceller=error, console, file
```
Comment (barsukovskii) [inline src/dist/cfg/log4j.properties]: error -> info

File: src/dist/cfg/log4j.properties [hunk 3]
```diff
@@ -6,6 +6,19 @@ log4j.appender.file.File=../log/server.log
...
+log4j.additivity.com.lykke.matching.engine.utils.order.MinVolumeOrderCanceller=false
+
+log4j.logger.com.lykke.matching.engine.utils.order.AllOrdersCanceller=error, console, file
```
Comment (barsukovskii) [inline src/dist/cfg/log4j.properties]: error -> info

File: src/dist/cfg/log4j.properties (hunk 1 above)
Comment (rocketmannot) [reply src/dist/cfg/log4j.properties]: fixed

File: src/dist/cfg/log4j.properties (hunk 2 above)
Comment (rocketmannot) [reply src/dist/cfg/log4j.properties]: fixed

File: src/dist/cfg/log4j.properties (hunk 3 above)
Comment (rocketmannot) [reply src/dist/cfg/log4j.properties]: fixed

**Pull request merged**